api/v1alpha1/clusterconfig_types.go

```diff
@@ -37,6 +37,12 @@ var DefaultDockerCertSANs = []string{
 	"host.docker.internal",
 }
 
+var DefaultNutanixCertSANs = []string{
+	"localhost",
+	"127.0.0.1",
+	"0.0.0.0",
+}
```

Comment:
nit: let's group all these vars.:

```suggestion
var (
	DefaultDockerCertSANs = []string{
		"localhost",
		"127.0.0.1",
		"0.0.0.0",
		"host.docker.internal",
	}

	DefaultNutanixCertSANs = []string{
		"localhost",
		"127.0.0.1",
		"0.0.0.0",
	}
)
```